config.json

```diff
@@ -65,6 +66,12 @@
       "strings",
       "control-flow (if-else statements)"
     ]
+  }, {
+    "difficulty": 1,
+    "slug": "grains",
+    "topics": [
+      "control-flow (loops)"
```

Comment:
You should add 'Bitwise operations' and possibly 'Performance' to the topics. The problem implies that you're supposed to attempt to optimize for performance.